readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: razorpay
 Tags: razorpay, payments, india, gravityforms, ecommerce
 Requires at least: 3.9.2
 Tested up to: 5.3.2
```

Comment:
Can you please update , tested upto version with latest WP after testing it.

Reply:
I had tested in 5.4 Will update the same.